VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.spec.ts

```diff
@@ -53,9 +68,13 @@ describe('UnreadMessagesComponent', () => {
 
     it('should init unread message count', fakeAsync(() => {
         const expectedCount = 5 * conference.participants.length;
-        component.unreadMessages = undefined;
+
+        const mappedUnreadMessages = mapper.mapUnreadMessageResponseArray(component.unreadMessages, conference.id);
+        unreadAdminMessageModelSpy.mapUnreadMessageResponseArray.and.returnValue(mappedUnreadMessages);
```

Comment:
By using an instance of the mapper this unit test is relying on that mapper as well. You should just define your own response as part of the test.

Reply:
This is why sonarcloud is showing 100% test coverage as you are touching those lines.

Reply:
That is part of a response-promise from the component. What alternatives do we have to circumvent that?